uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/staff-leave-report.html

```diff
@@ -91,8 +92,9 @@
                               </td>
                               <td></td>
                               <td></td>
-                              <td ng-repeat="absenceType in report.absenceTypesFiltered track by $index">
-                                {{(entry.type_id == absenceType.id ? (entry.amount | number:0) : '')}}
+                              <td ng-repeat="absenceType in report.absenceTypesFiltered track by $index"
+                                title="({{ absenceType.calculation_unit_label | lowercase }})">
+                                {{(entry.type_id == absenceType.id ? +(entry.amount | number:0) + absenceType.calculation_unit_symbol : '')}}
```

Comment:
why `+(entry.amount | number:0)` and not just `(entry.amount | number:0)`? I also don't think the `number` filter mind you passing a string to it.

Reply:
Was just an automatic action. I added `+` everywhere. Now they are removed.